guides/sev.md

```diff
@@ -58,20 +58,20 @@ sudo ./sevctl/target/debug/sevctl export --full /opt/sev/cert_chain.cert
 
 By default, the `kata-qemu-sev` runtime class uses pre-attestation with the
 `online-sev-kbc` and [simple-kbs](https://github.com/confidential-containers/simple-kbs) to attest the guest and provision secrets.
-`simple-kbs` is a basic prototype key broker that can validate a guest measurement according to a policy and conditionally release secrets.
+`simple-kbs` is a basic prototype key broker which can validate a guest measurement according to a specified policy and conditionally release secrets.
 To use encrypted images, signed images, or authenticated registries with SEV, you should setup `simple-kbs`.
 If you simply want to run an unencrypted container image, you can disable pre-attestation by adding the following annotation
 `io.katacontainers.config.pre_attestation.enabled: "false"` to your pod.
 
-If you are using pre-attestation, you will need to add an annotation to your pod that contains the URI of `simple-kbs`.
+If you are using pre-attestation, you will need to add an annotation to your pod configuration which contains the URI of a `simple-kbs` instance.
 This annotation should be of the form `io.katacontainers.config.pre_attestation.uri: "<KBS IP>:44444"`.
-Port 44444 is the default port per the directions below, but it can be configured.
+Port 44444 is the default port per the directions below, but it may be configured to use another port.
 The KBS IP must be accessible from inside the guest.
 Usually it should be the public IP of the node where `simple-kbs` runs.
 
 The SEV policy can also be set by adding `io.katacontainers.config.sev.policy: "<SEV POLICY>"` to your pod configuration.
 Setting the second bit of the policy enables SEV-ES.
-For more information see chapter 3 of the AMD Secure Encrypted Virtualization API.
+For more information see chapter 3 of the [Secure Encrypted Virtualization API](https://www.amd.com/system/files/TechDocs/55766_SEV-KM_API_Specification.pdf#page=31).
```

Comment:
I haven't seen a license like this on other things in open source. Can this be updated to something more open source conventional?

Reply:
It's not that the spec needs to be "open source" it's just that if there's something like a paywall in front of a spec it's not friendly for developers to access. In this case, there's an unusual license to accept to look at the document. Maybe there's a friendlier API specification to point to rather than that one?

Reply:
Maybe y'all could republish it with an open source friendly license? I think most specs just have copyright and some disclaimer. (not a lawyer)

Reply:
So I can follow the link https://www.amd.com/system/files/TechDocs/55766_SEV-KM_API_Specification.pdf#page=31 without needing to accept anything?
Though the first page of the spec which requires the reader to acknowledge a specification agreement before reading further concerns me? (Though I was only checking the link not going on to read the document anyway)

I do share @dcmiddle license concerns, But if there was a requirement to acknowledge something in order to download (more a friendly to use concern)  -  Now there isn' acknowledgement to download, but there is still an specification agreement on the first page that perhaps worries me more? Is there a risk that someone working on other TEE's could inadvertently click on link from CoCo docs and end up with a specification document that could cause legal headaches?

Reply:
Perhaps remove link, lets merge this PR and put the question to the community regarding concerns on accessing specs?

Reply:
That makes sense to me. The rest of the PR is great and it would be better to see it merged than waiting indefinitely on this point. (Unless Larry is close to resolution on the licensing).

Reply:
fwiw I really see this as a non-issue. I think the legalese is pretty standard. Our work doesn't come close to violating any of the clauses and people who click on the link can decide for themselves if they want to read the spec. I think there is basically no chance that AMD will adjust the license. If we can't get past this then I guess removing the link is an easy option, but I think there is really nothing to worry about.